sos/report/plugins/ubuntu.py

```diff
@@ -48,12 +48,18 @@ def setup(self):
                "/var/log/ubuntu-advantage.log",
                "/var/log/ubuntu-advantage.log.1",
                "/var/log/ubuntu-advantage.log.2*",
+               "/var/log/ubuntu-advantage-apt-hook.log",
+               "/var/log/ubuntu-advantage-apt-hook.log.1",
+               "/var/log/ubuntu-advantage-apt-hook.log.2*",
                "/var/log/ubuntu-advantage-timer.log",
                "/var/log/ubuntu-advantage-timer.log.1",
                "/var/log/ubuntu-advantage-timer.log.2*",
             ])
         else:
-            self.add_copy_spec("/var/log/ubuntu-advantage.log*")
-            self.add_copy_spec("/var/log/ubuntu-advantage-timer.log*")
+            self.add_copy_spec([
+               "/var/log/ubuntu-advantage.log",
+               "/var/log/ubuntu-advantage-apt-hook.log",
+               "/var/log/ubuntu-advantage-timer.log",
```

Comment:
Is it intentional to drop the `*` for collecting rotated logs?

Reply:
wow, a typo, good catch